mscore/palette.cpp

```diff
@@ -695,6 +728,36 @@ void Palette::applyPaletteElement(Element* element, Qt::KeyboardModifiers modifi
                         score->cmdAddSpanner(spanner, i, startSegment, endSegment);
                         }
                   }
+            else if (element->isDynamic() || element->isStaffText() || element->isTempoText() || element->isSystemText()) {
+                  int track1 = sel.staffStart() * VOICES;
+                  int track2 = sel.staffEnd() * VOICES;
+                  Segment* startSegment = sel.startSegment();
+                  Segment* endSegment = sel.endSegment(); //keep it, it could change during the loop
+                  for (Segment* s = startSegment; s && s != endSegment; s = s->next1()) {
+                        QSet<Staff*> set;
+                        for (int track = track1; track < track2; ++track) {
+                              Element* e = s->element(track);
+                              if (e == 0 || !score->selectionFilter().canSelect(e) || !score->selectionFilter().canSelectVoice(track))
+                                    continue;
+                              if (e->isChordRest()) {
+                                    if (!set.contains(e->staff())) {
+                                          applyDrop(score, viewer,e->isChord() ? toChord(e)->upNote() : e, element, modifiers);
+                                          if (element->isTempoText() || element->isSystemText())
```

Comment:
Here though you're not using braces, which are not strictly needed here, but might be better as there are 2 lines (even if one is just a comment

Reply:
Maybe you could move the comment to above of after the `if`?

Reply:
I tried both options but, personally, I found it was clearer having the comments in the blocks.
And for me it wasn't that clear whether comment were counted as line in the context of "Exception 2". Now it is clear :-)

Reply:
Added braces to include the comment in the block.